parlai/core/torch_generator_agent.py

```diff
@@ -313,6 +329,18 @@ def add_cmdline_args(cls, argparser):
             help='Skip beam search. Useful for speeding up training, '
             'if perplexity is the validation metric.',
         )
+        agent.add_argument(
+            '--inference',
+            choices={'beam', 'greedy', 'topk', 'nucleus', 'topp'},
```

Comment:
why do we have `topp` AND `nucleus` in here?

Reply:
Since people call it both, I figured it'd be nice to give it the alias. You suggest only one I take it?

Reply:
I found it confusing, I think people reading this would wonder what the difference was. Maybe add a comment saying they are the same

Reply:
I'm just going to only keep one.